server/authn/oidc/oidc.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/MicahParks/keyfunc"
 	"github.com/go-errors/errors"
 	"github.com/golang-jwt/jwt/v4"
-	grpcAuth "github.com/grpc-ecosystem/go-grpc-middleware/auth"
+	grpc_auth "github.com/grpc-ecosystem/go-grpc-middleware/auth"
```

Comment:
Why this change?

Reply:
For a subsequent PR can we add a linting rule to prevent this from happening and normalize?